.github/workflows/android.yaml

```diff
@@ -0,0 +1,20 @@
+name: Build the Android files from NextJS
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Java from Ubuntu repos
+      run: sudo apt update && sudo apt dist-upgrade -y && sudo apt install openjdk-11-jdk -y
+    - name: Execute the build script
+      run: ./build_android.sh
+    - uses: actions/upload-artifact@v2
+      with:
+        name: android-debug-apk
+        path: rogue-thi-app/android/app/build/outputs/apk/debug/app-debug.apk
```

Comment:
Is the debug apk simply unsigned? Or is it actually built with debug symbols / in debugging mode? In the latter case we might want to also upload the release APK as an artifact?

Reply:
The debug APK is signed with the debug key, i.e. nothing trustworthy. I don't expect it to have debug symbols or something similar, though.